options/options.go

```diff
@@ -240,3 +247,21 @@ func validateFormat(format string) bool {
 		return false
 	}
 }
+
+// isCheckValidForLocalRepo returns an error if the check is not valid for local repository.
+func (o *Options) isCheckValidForLocalRepo() error {
```

Comment:
Nit: maybe name the fn `isValidForLocalRepo` since it is validating the `Options` struct and not a check.

Reply:
Yep, agreed. Took care of it.